pkg/termination/write_test.go

```diff
@@ -66,3 +66,29 @@ func TestExistingFile(t *testing.T) {
 		}
 	}
 }
+
+func TestMaxSizeFile(t *testing.T) {
+	value := ""
+	for n := 0; n < 4096; n++ {
+		value += "a"
+	}
+	tmpFile, err := ioutil.TempFile(os.TempDir(), "tempFile")
+	if err != nil {
+		log.Fatal("Cannot create temporary file", err)
+	}
+	// Remember to clean up the file afterwards
+	defer os.Remove(tmpFile.Name())
+
+	logger, _ := logging.NewLogger("", "test termination")
+	defer func() {
+		_ = logger.Sync()
+	}()
```

Comment:
I'm not sure this is required - what happens if it's removed?

Reply:
Removed.